vault/logical_system.go

```diff
@@ -265,6 +265,10 @@ func NewSystemBackend(core *Core) *SystemBackend {
 						Type:        framework.TypeCommaStringSlice,
 						Description: strings.TrimSpace(sysHelp["tune_audit_non_hmac_response_keys"][0]),
 					},
+					"options": &framework.FieldSchema{
+						Type:        framework.TypeMap,
```

Comment:
If you use TypeKVPair for this, you will be ensured a map[string]string and would save the extra checking below and give some extra CLI support.